tensorboard/context.py

```diff
@@ -32,9 +32,14 @@ class RequestContext(object):
 
     Fields:
       auth: An `AuthContext`, which may be empty but is never `None`.
+      remote_ip: An `ipaddress.IPv4Address` or `ipaddress.IPv6Address`, which may be None.
+        Best guess of the IP Address of the end user.
+      x_forwarded_for: A list of `ipaddress.IPv4Address` or `ipaddress.IPv6Address`,
```

Comment:
Contexts are meant to be immutable. Should this be a tuple instead?
Either that or shallow-copy the list in both the initializer and the
`@property` getter, but just using a tuple seems cleaner.

Reply:
Done for real!